api/swagger.yaml

```diff
@@ -2725,6 +2725,11 @@ definitions:
             description: "Run an init inside the container that forwards signals and reaps processes. This field is omitted if empty, and the default (as configured on the daemon) is used."
             type: "boolean"
             x-nullable: true
+          Sysctls:
```

Comment:
Can you update the API version history, and add a bullet for this new option? https://github.com/moby/moby/blob/master/docs/api/version-history.md#v139-api-changes

(make sure to mention all of `POST /services/create`, `POST  /services/{id}/update`, and `GET /services/{id}`)

Reply:
Ah I always forget the API version history.